deployment/aws-python-storage-stack-automation/storage_stack_lifecycle.yaml

```diff
@@ -1,15 +1,15 @@
 AWSTemplateFormatVersion: 2010-09-09
-Description: "Automatically deploy File Storage Security - Storage stack to every new S3 bucket created and register it to the defined Scanner Stack."
+Description: "FSS Bucket Lifecycle Automation Stack. Deploy FSS Storage to each new S3 bucket. When S3 Bucket is deleted FSS stack is removed if monitored"
```

Comment:
Description: "FSS Bucket Lifecycle Automation Stack. Deploy FSS Storage to each new S3 bucket. When the S3 Bucket is deleted, the FSS stack is removed, if monitored."

Reply:
Updated